docs/source/introduction.rst

```diff
@@ -150,6 +161,101 @@ installed, which
+
+**CF data model**
+-----------------
+
+The CF (Climate and Forecast) metadata conventions
+(http://cfconventions.org) provide a description of the physical
+meaning of data and of their spatial and temporal properties and are
+designed to promote the creation, processing, and sharing of climate
+and forecasting data using netCDF files and libraries
+(https://www.unidata.ucar.edu/software/netcdf).
+
+`The CF data model
+<https://cfconventions.org/cf-conventions/cf-conventions.html#appendix-CF-data-model>`_
+identifies the fundamental elements ("constructs") of the CF
+conventions and shows how they relate to each other, independently of
+the netCDF encoding.
+
+The CF data model defines a **field construct** for storing data with
+all of its metadata. It is defined in CF-|version| as follows:
+
+.. glossary::
+
+  field construct
+    corresponds to a CF-netCDF data variable with all of its
+    metadata. It consists of
+
+    - descriptive properties that apply to field construct as a whole
+      (e.g. the standard name),
+    
+    - a data array,
+
+    - a **domain construct** that describes the locations of each cell
+      of the data array (i.e. the "domain"),
+    
+    - **metadata constructs** that describe the physical nature of the
+      data array, defined by
+
+      .. glossary::
+         
+        field ancillary constructs
+          corresponding to CF-netCDF ancillary variables
+      
+        cell method constructs
+          corresponding to a CF-netCDF cell_methods attribute of data
+          variable
+
+  domain construct
+    that describes the locations of each cell of the domain. It may
+    exist independently of a **field construct** and consists of
+
+    - descriptive properties that apply to domain construct as a whole,
+    
+    - **metadata constructs** that describe the locations of each cell
+      of the domain, defined by 
+    
+    .. glossary::
+         
+      domain axis constructs
+        corresponding to CF-netCDF dimensions or scalar coordinate
+        variables
+    
+      dimension coordinate constructs
+        corresponding to CF-netCDF coordinate variables or numeric
+        scalar coordinate variables
+    
+      auxiliary coordinate constructs
+        corresponding to CF-netCDF auxiliary coordinate variables and
+        non-numeric scalar coordinate variables
+    
+      coordinate reference constructs
+        corresponding to CF-netCDF grid mapping variables or the
+        formula_terms attribute of a coordinate variable
+    
+      domain ancillary constructs
+        corresponding to CF-netCDF variables named by the
+        formula_terms attribute of a coordinate variable
+    
+      cell measure constructs
+        corresponding to CF-netCDF cell measure variables
+
+      domain topology constructs
+        corresponding to CF-netCDF UGRID mesh topology variables
+
+      cell connectivity constructs
+        corresponding to CF-netCDF UGRID connectivity variables
```

Comment:
Numerous alignment issues here which break the formatting (see from a 'view file' on the GH UI else in an RST previewer/validator). I fixed it up locally, this should be what I think it intended. I also think adding leading colons to introduce the lists makes it clearer so I've put those in too:

```suggestion
.. glossary::

   field construct
     corresponds to a CF-netCDF data variable with all of its
     metadata. It consists of:

     - descriptive properties that apply to field construct as a whole
       (e.g. the standard name),

     - a data array,

     - a **domain construct** that describes the locations of each cell
       of the data array (i.e. the "domain"),

     - **metadata constructs** that describe the physical nature of the
       data array, defined by:

       .. glossary::

          field ancillary constructs
            corresponding to CF-netCDF ancillary variables

          cell method constructs
            corresponding to a CF-netCDF cell_methods attribute of data
            variable

          domain construct
            that describes the locations of each cell of the domain. It
            may exist independently of a **field construct** and
            consists of:

            - descriptive properties that apply to domain construct as
              a whole,

            - **metadata constructs** that describe the locations of each cell
              of the domain, defined by:

              .. glossary::

                 domain axis constructs
                   corresponding to CF-netCDF dimensions or scalar coordinate
                   variables

                 dimension coordinate constructs
                   corresponding to CF-netCDF coordinate variables or numeric
                   scalar coordinate variables

                 auxiliary coordinate constructs
                   corresponding to CF-netCDF auxiliary coordinate variables and
                   non-numeric scalar coordinate variables

                 coordinate reference constructs
                   corresponding to CF-netCDF grid mapping variables or the
                   formula_terms attribute of a coordinate variable

                 domain ancillary constructs
                   corresponding to CF-netCDF variables named by the
                   formula_terms attribute of a coordinate variable

                 cell measure constructs
                   corresponding to CF-netCDF cell measure variables

                 domain topology constructs
                   corresponding to CF-netCDF UGRID mesh topology variables

                 cell connectivity constructs
                   corresponding to CF-netCDF UGRID connectivity variables
```

Reply:
Hi Sadie - not sure what was wrong with the original - it renders fine when built with sphinx ...

Reply:
Aha, I now realise I made the mistake of viewing it in markdown and not RST :facepalm:. So that explains it - filing this under non-existent issues, please ignore. Sorry, marking as resolved.